components/TextFields/src/MDCTextInputController.h

```diff
@@ -129,34 +129,42 @@
 
  The underline is an overlay that can be hidden depending on the editing state of the input text.
 
- Default is UITextFieldViewModeAlways.
+ Default is underlineViewModeDefault.
  */
 @property(nonatomic, assign) UITextFieldViewMode underlineViewMode;
 
 /**
- Default value for above property.
+ Default value for underlineViewMode.
 
- Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ Default is UITextFieldViewModeAlways.
  */
 @property(class, nonatomic, assign) UITextFieldViewMode underlineViewModeDefault;
 
-/** Color the underline changes to when the input is editing. */
+/** 
+ Color the underline changes to when the input is editing. 
+ 
+ Default is underlineColorActiveDefault.
+ */
 @property(nonatomic, null_resettable, strong) UIColor *underlineColorActive;
 
 /**
- Default value for above property.
+ Default value for underlineColorActive.
 
- Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ Default is primary color or blue.
  */
 @property(class, nonatomic, null_resettable, strong) UIColor *underlineColorActiveDefault;
 
-/** Color of the underline when the input is not editing but still enabled. */
+/** 
+ Color of the underline when the input is not editing but still enabled. 
+ 
+ Default is underlineColorNormalDefault.
+ */
 @property(nonatomic, null_resettable, strong) UIColor *underlineColorNormal;
 
 /**
- Default value for above property.
+ Default value for underlineColorNormal.
 
- Class property. Replaces UIAppearance for classes that do not inherit from UIView.
+ Default is same as inlinePlaceholderColorDefault.
```

Comment:
Maybe just state the default without making the reader jump around to find the inlinePlaceholder one?